lib/chai/core/assertions.js

```diff
@@ -968,19 +1492,32 @@ module.exports = function (chai, _) {
   });
 
   /**
-   * ### .instanceof(constructor)
+   * ### .instanceof(constructor[, msg])
```

Comment:
Maybe we could add some text detailing that this fails whenever not passed a valid constructor instance and which criteria are used to determine whether something is valid or not.